web_src/css/base.css

```diff
@@ -793,6 +793,16 @@ a.label,
   user-select: auto;
 }
 
+.ui.vertical.menu .item .menu .item {
+  color: var(--color-text-light-2);
+  text-indent: 16px;
+}
```

Comment:
Here and below:

Are we sure these are general styles for whole Gitea? If yes, keep them, otherwise, use Gitea's own class names.

Reply:
Not sure yet. Added a commit to use custom class https://github.com/go-gitea/gitea/pull/24200/commits/3d1def213f629a2422bdddf251b2c314888d1516